docs/source/containerization/kubernetes.mdx

```diff
@@ -1,246 +1,274 @@
+        router:
+          timeout: 6min
+        all:
+          experimental_enable_http2: false
+          timeout: 5min
+```
+
+Support for `@defer` is unsupported in gateway, so it needs to be disabled in the router.
+
+```yaml title="values.yaml"
+router:
+  router:
+    configuration:
+      supergraph:
+        defer_support: false
 ```
 
-Once executed, you can check the status of the helm deploy:
+The gateway propagates subgraph errors to clients, but the router doesn't by default, so it needs to be configured to propagate subgraph errors.
+
+```yaml title="values.yaml"
+router:
+  router:
+    configuration:
+      include_subgraph_errors:
+        all: true
+```
+
+## Deploy router with Helm
+
+### Install Helm
+
+* Install [Helm](https://helm.sh/docs/intro/install/) **version 3**. The Apollo Router's Helm chart requires Helm v3.
+
+> ⚠️ Your Kubernetes version must be compatible with Helm v3. For details, see [Helm Version Support Policy](https://helm.sh/docs/topics/version_skew/#supported-version-skew).
+
+### Set up cluster
+
+Install the tools and provision the infrastructure for your Kubernetes cluster.
+
+> For example, see the [Setup from Apollo's Build a Supergraph tutorial](https://github.com/apollosolutions/build-a-supergraph/tree/main/01-setup#01---setup). It provides steps you can reference for gathering accounts and credentials for your cloud platform (GCP or AWS), provisioning resources, and deploying your subgraphs.
+
+(TODO: refine the steps between this #set-up-cluster and the next #set-up-self-hosted-router)
+
+### Set up self-hosted router
+
+Set up a self-hosted router by following the [first six steps of the self-hosted router quickstart](/graphos/quickstart/self-hosted), and stopping when you reach step [7. Deploy your router and connect clients](https://www.apollographql.com/docs/graphos/quickstart/self-hosted/#7-deploy-your-router-and-connect-clients).
+
+
+## Install router Helm chart
+
+You can install the router chart from either a local repository or the container registry.
+
+### Install chart from container registry
+
+* Login to the container registry by following the guide to [authenticating to the GitHub container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
+
+* After logging in, verify your access to the registry by showing the latest router chart from `helm`:
+
+  ```bash
+  helm show values oci://ghcr.io/apollographql/helm-charts/router
+  ```
+
+* Install the OCI image from the GiHub container registry:
+
+  ```bash title="OCI registry"
+  helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.8 --values router/values.yaml
```

Comment:
This needs some refinement. e.g.: prometheus we don't want to set and the multiple values files for common and env are missing.